nautobot/extras/templates/extras/team_retrieve.html

```diff
@@ -0,0 +1,71 @@
+{% extends 'generic/object_retrieve.html' %}
+{% load helpers %}
+
+{% block content_left_page %}
+<div class="panel panel-default">
+    <div class="panel-heading">
+        <strong>Team</strong>
+    </div>
+    <table class="table table-hover panel-body attr-table">
+        <tr>
+            <td>Name</td>
+            <td>{{ object.name }}</td>
+        </tr>
+        <tr>
+            <td>Phone</td>
+            <td>
+                {% if object.phone %}
+                    <a href="tel:{{ object.phone }}">{{ object.phone }}</a>
+                {% else %}
+                    <span class="text-muted">&mdash;</span>
+                {% endif %}
```

Comment:
Maybe a future enhancement, but the `phone` and `email` rendering logic here, below, and elsewhere might be a great candidate for implementing as templatetags?

Reply:
https://github.com/nautobot/nautobot/issues/5107